src/app/modules/shared/components/breadcrumb/breadcrumb.component.ts

```diff
@@ -1,14 +1,21 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { Breadcrumb } from './breadcrumb.model';
+import { BreadcrumbService } from './breadcrumb.service';
 
 @Component({
   selector: 'app-breadcrumb',
   templateUrl: './breadcrumb.component.html',
 })
-export class BreadcrumbComponent implements OnInit {
+export class BreadcrumbComponent {
+  public breadcrumbs: Breadcrumb[] = [];
 
-  constructor() { }
-
-  ngOnInit(): void {
+  constructor(private breadcrumbService: BreadcrumbService) {
```

Comment:
Doing subscriptions in the constructor can often result in bugs. We should always utilize the onInit lifecycle method. Additionally, we don't even need to subscribe at all if we use the `async` angular pipe in the template